ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs

```diff
@@ -239,7 +240,15 @@ public override void OnReactInstanceDispose()
             try
             {
 #if WINDOWS_UWP
-                var storageFile = await StorageFile.GetFileFromPathAsync(uri.LocalPath).AsTask().ConfigureAwait(false);
+                StorageFile storageFile;
+                if (FutureAccessList.IsValidURI(uri.ToString()))
+                {
+                    storageFile = await FutureAccessList.GetFileAsync(uri.ToString());
```

Comment:
Why use `ConfigureAwait(false)` below but not here?

Reply:
I have fixed it yesterday, after someone else pointed it to me